settings/src/main/scala/org/typelevel/sbt/TypelevelSettingsPlugin.scala

```diff
@@ -29,13 +29,17 @@ object TypelevelSettingsPlugin extends AutoPlugin {
   object autoImport {
     lazy val tlFatalWarnings =
       settingKey[Boolean]("Convert compiler warnings into errors (default: false)")
+    lazy val tlJdkRelease =
+      settingKey[Option[Int]](
+        "JVM target version for the compiled bytecode, None results in default scalac and javac behavior (no --release flag is specified). (default: Some(8), supported values: 8, 9, 10, 11, 12, 13, 14, 15, 16, 17)")
   }
 
   import autoImport._
   import TypelevelKernelPlugin.autoImport._
 
   override def globalSettings = Seq(
     tlFatalWarnings := false,
+    tlJdkRelease := Some(8),
```

Comment:
Not to be too annoying, but we can make this `None` and get this into `series/0.4` :)